cli/src/pcluster/config/cluster_config.py

```diff
@@ -2101,6 +2132,25 @@ def multi_az_enabled(self):
         """Return true if more than one AZ are defined in the queue Networking section."""
         return len(self.networking.az_list) > 1
 
+    def get_gpu_health_checks_enabled_keys(self) -> List[str]:
```

Comment:
what's this method for? If it used only for testing, it should be moved on testing utils class